pm_uep_opt_write/test-data/tree-default-folder.txt

```diff
@@ -1,92 +1,223 @@
- extracting: muon-airss-out/uep/Si_17/Si_17.uep  
-  inflating: muon-airss-out/uep/Si_18/Si_18.yaml  
- extracting: muon-airss-out/uep/Si_18/Si_18.uep  
-  inflating: muon-airss-out/uep/Si_19/Si_19.yaml  
- extracting: muon-airss-out/uep/Si_19/Si_19.uep  
-  inflating: muon-airss-out/uep/Si_20/Si_20.yaml  
- extracting: muon-airss-out/uep/Si_20/Si_20.uep  
-  inflating: muon-airss-out/uep/Si_21/Si_21.yaml  
- extracting: muon-airss-out/uep/Si_21/Si_21.uep  
-  inflating: muon-airss-out/uep/Si_22/Si_22.yaml  
- extracting: muon-airss-out/uep/Si_22/Si_22.uep  
-  inflating: muon-airss-out/uep/Si_23/Si_23.yaml  
- extracting: muon-airss-out/uep/Si_23/Si_23.uep  
-  inflating: muon-airss-out/uep/Si_24/Si_24.yaml  
- extracting: muon-airss-out/uep/Si_24/Si_24.uep  
-  inflating: muon-airss-out/uep/Si_25/Si_25.yaml  
- extracting: muon-airss-out/uep/Si_25/Si_25.uep  
-  inflating: muon-airss-out/uep/Si_26/Si_26.yaml  
- extracting: muon-airss-out/uep/Si_26/Si_26.uep  
-  inflating: muon-airss-out/uep/Si_27/Si_27.yaml  
- extracting: muon-airss-out/uep/Si_27/Si_27.uep  
-  inflating: muon-airss-out/uep/Si_28/Si_28.yaml  
- extracting: muon-airss-out/uep/Si_28/Si_28.uep  
-  inflating: muon-airss-out/uep/Si_29/Si_29.yaml  
- extracting: muon-airss-out/uep/Si_29/Si_29.uep  
-  inflating: muon-airss-out/uep/Si_30/Si_30.yaml  
- extracting: muon-airss-out/uep/Si_30/Si_30.uep  
-  inflating: muon-airss-out/uep/Si_31/Si_31.yaml  
- extracting: muon-airss-out/uep/Si_31/Si_31.uep  
-  inflating: muon-airss-out/uep/Si_32/Si_32.yaml  
- extracting: muon-airss-out/uep/Si_32/Si_32.uep  
-  inflating: muon-airss-out/uep/Si_33/Si_33.yaml  
- extracting: muon-airss-out/uep/Si_33/Si_33.uep  
-  inflating: muon-airss-out/uep/Si_34/Si_34.yaml  
- extracting: muon-airss-out/uep/Si_34/Si_34.uep  
-  inflating: muon-airss-out/uep/Si_35/Si_35.yaml  
- extracting: muon-airss-out/uep/Si_35/Si_35.uep  
-  inflating: muon-airss-out/uep/Si_36/Si_36.yaml  
- extracting: muon-airss-out/uep/Si_36/Si_36.uep  
-  inflating: muon-airss-out/uep/Si_37/Si_37.yaml  
- extracting: muon-airss-out/uep/Si_37/Si_37.uep  
-  inflating: muon-airss-out/uep/Si_38/Si_38.yaml  
- extracting: muon-airss-out/uep/Si_38/Si_38.uep  
-  inflating: muon-airss-out/uep/Si_39/Si_39.yaml  
- extracting: muon-airss-out/uep/Si_39/Si_39.uep  
-  inflating: muon-airss-out/uep/Si_40/Si_40.yaml  
- extracting: muon-airss-out/uep/Si_40/Si_40.uep  
-  inflating: muon-airss-out/uep/Si_41/Si_41.yaml  
- extracting: muon-airss-out/uep/Si_41/Si_41.uep  
-  inflating: muon-airss-out/uep/Si_42/Si_42.yaml  
- extracting: muon-airss-out/uep/Si_42/Si_42.uep  
-  inflating: muon-airss-out/uep/Si_43/Si_43.yaml  
- extracting: muon-airss-out/uep/Si_43/Si_43.uep  
-  inflating: muon-airss-out/uep/Si_44/Si_44.yaml  
- extracting: muon-airss-out/uep/Si_44/Si_44.uep  
-  inflating: muon-airss-out/uep/.collection  
-  inflating: params.yaml             
-  inflating: input.cell              
+muon-airss-out
```

Comment:
To keep test runtime down, the number of configurations in this dataset should also be reduced by changing poisson_r, just like the other one.

Reply:
changed